src/utils/determineTemporalLabel.js

```diff
@@ -12,5 +12,8 @@ import {
 export const determineTemporalLabel = (timestamp) => {
   const date = new Date(timestamp)
 
+  // If the date is invalid, return an empty string
```

Comment:
Does this need a testcase in determineTemporalLabel if the timestamp is invalid?